frontend/src/components/cards/authCard/AuthCard.tsx

```diff
@@ -55,28 +56,6 @@ export default function AuthCard(props: iAuthCard) {
     }
 
 
-    // TODO: Make this into a react hook
-    const handleFormValidation = (input_name: string, error_type: string) => {
-        // console.log("error msg: " + input_name + " " + "error type:" + error_type);
```

Comment:
Why is this comment still here?

Reply:
to uncomment later for console logging inputs